src/api/status/package.json

```diff
@@ -17,7 +17,9 @@
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
     "@senecacdot/satellite": "^1.x",
+    "cross-fetch": "^3.1.4",
```

Comment:
We don't need this, you can use `fetch` from Satellite: https://github.com/Seneca-CDOT/satellite#fetch

```js
const { fetch } = require('@senecacdot/satellite');
```

Reply:
Changed